### DISCUSSION TITLE
[Ineligible for community support] Cannot route message through exchange

### DISCUSSION THREAD

**Discussion by ko2024** [rabbitmq/rabbitmq-server#10596] [Other] [answered]
I had a case where a master node restarted and the service lost synchronization and I noticed that it started generating _Cannot route message through exchange_ events and it was not possible to publish or consume the messages. Has anyone already gotten this report in version 3.8.12? 

Error:
 channel 1:
operation basic.publish caused a channel exception precondition_failed: Cannot route message through exchange

**Comment by lukebakken** [comment, marked as answer]
RabbitMQ 3.8, 3.9, 3.10 and (soon) 3.11 are all unsupported. If you can reproduce this error on a _supported_ version of RabbitMQ, let us know.
